dso/model.py

```diff
@@ -13,6 +13,18 @@
 import unixutil.model
 
 
+class ComponentContext(enum.StrEnum):
+    SAP_INTERNAL = 'sap-internal'
+    PUBLIC = 'public'
+
+
+class SastStatus(enum.StrEnum):
+    LOCAL_LINTING = 'local-linting'
+    NO_LINTING = 'no-linting'
+    CENTRAL_LINTING = 'central-linting'
+    CENTRAL_AND_LOCAL_LINTING = 'central-and-local-linting'
```

Comment:
I'm not sure if we should really use extra enum values for combinations of other values, this can become quite cumbersome I'd imagine if we liked to add other statuses in the future. What about allowing `SastFinding` either a `SastStatus` or a list of `SastStatus`?